### ISSUE TITLE
Fixed aspect ratio possibility to any (picture containing) widget

### ISSUE THREAD

**Issue by mgustavs** [elementor/elementor#7758]
<#5857 touches, but not quite >

## Prerequisites
- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

I've been breaking my brain over this for days here. I need pictures that change on hover (to another pic) with links. My client needs these pictures to be visible in the original ratio (9:4) on any device width - 320 up to 1920 wide. I have sniffed every widget I know on Pro and I can't achieve this! If I just put a photo, there is a ratio and I can add a link, but no other picture on hover. Whatever else I choose - call to action | flip-box | column (with added link) etc. - they all do have different aspect ratios on different screens (set the min height or I don't), because they all are certain height restricted... And all given background display options tweak the photo to their needs (or display halfway) and there is no way to tell widgets to have same ratio as the added background photo. Or just a fixed ratio! To be honest I have come across this limitation many times while making any page responsive and it does frustrate. Any gallery, any picture has fixed ratio, but none of the action widgets...

To summarize - all widgets (with possible picture inserted) having an option to set a certain aspect ratio across all screens would be just great! (one little box where to add a number from 0,01 to 1, say + value for > : < and - for < : > )

**Comment by mgustavs** [comment]
...I've come around this adding sophisticated Sliders with layers and their animations (Smart Slider 3), but it is just bass-ackwords..

**Comment by bainternet** [comment]
Currently, this feature is not supported, but we will look into it in the future if we see a large demand for it.

Thanks for the feedback.

**Comment by mgustavs** [comment]
Still missing it...   That bunch of sliders is slowing the page down for at least 3 seconds (!) ...

**Comment by jp70m** [comment]
Fixed aspect Ratio for the Flip Box

**Comment by Lovely-Fellow** [comment]
Needed for me, too

**Comment by mcj777** [comment]
+1

**Comment by marcieee73** [comment]
+ 1

**Comment by yachal-upson** [comment]
+1 Oh my, how is this not a feature? 
I just.. want.. my.. images (or widgets full stop).. to.. scale.. responsively: while keeping their aspect ratio. Please

**Comment by bg1137** [comment]
I have made this work in elementor by using a custom height calc off the vw.
